common/lib/xmodule/xmodule/tests/test_course_metadata_utils.py

```diff
@@ -230,4 +233,44 @@ def nop_gettext(text):
         # we still need to test it with a bad format string in order to
         # satisfy the coverage checker.
         with self.assertRaises(ValueError):
-            mock_strftime_localized(test_datetime, 'BAD_FORMAT_SPECIFIER')
+            self.mock_strftime_localized(test_datetime, 'BAD_FORMAT_SPECIFIER')
+
+    @freeze_time("2016-03-27 00:59:00")
+    def test_time_zone_before(self):
+        """
+        Test correctly formatted datetime is returned when time zone is specified
+        and before daylight savings time has started
+        """
+        time_zone_before_parsable = "2016-03-27 00:59:00"
+        time_zone_before_datetime = datetime(2016, 03, 27, 00, 59, 00, tzinfo=utc)
+
+        actual_start_return = course_start_datetime_text(DEFAULT_START_DATE, time_zone_before_parsable,
+                                                         'DATE_TIME', timezone('Europe/Paris'),
+                                                         self.nop_gettext, self.mock_strftime_localized)
+        expected_start_return = "DATE_TIME " + "2016-03-27 01:59:00 CET"
+        self.assertEqual(actual_start_return, expected_start_return)
+
+        actual_end_return = course_end_datetime_text(time_zone_before_datetime, 'TIME',
+                                                     timezone('Europe/Paris'), self.mock_strftime_localized)
+        expected_end_return = "TIME " + "2016-03-27 01:59:00 CET"
+        self.assertEqual(actual_end_return, expected_end_return)
+
+    @freeze_time("2016-03-27 01:00:00")
+    def test_time_zone_after(self):
```

Comment:
I don't know how hard this would be - but could there be a test that includes a DST begin/end period, and ensures the time zone conversion still functions properly?